source/crud/crud.rst

```diff
@@ -12,7 +12,7 @@ Driver CRUD API
 :Status: Approved
 :Type: Standards
 :Minimum Server Version: 2.4
-:Last Modified: Aug. 5, 2016
+:Last Modified: Aug. 17, 2016
```

Comment:
@craiggwilson: Shall I bump this to the current data?

Based on [`crud.rst`'s git history](https://github.com/mongodb/specifications/commits/master/source/crud/crud.rst), it was last bumped with @estolfo's first commit for DRIVERS-291 (collation options).